meta/labs/meetings/15-09-03_who_writes_the_initial_playbook.md

```diff
@@ -0,0 +1,69 @@
+# Who Writes The Initial Draft of The Playbook?
```

Comment:
Can you also use proper title case formatting?

http://www.grammar-monster.com/lessons/capital_letters_title_case.htm